packages/x-data-grid/src/components/toolbarV8/Toolbar.tsx

```diff
@@ -1,6 +1,6 @@
 import * as React from 'react';
```

Comment:
Will add tests for the new toolbar in a follow-up PR to avoid this one growing any more.